theme/table.css

```diff
@@ -3,28 +3,31 @@
  * For licensing, see LICENSE.md.
  */
 
-.ck.ck-content table {
-	/* Give the table some air and center it horizontally */
+.ck.ck-content figure.table {
```

Comment:
002n is enough according to our standards (e.g. `.ck-content .image {}`) so `.ck-content .table` should be OK.

Reply:
In fact, because of that, the table styles override the generic nested editable styles and we've lost the blue border around it when the cell is focused.

![image](https://user-images.githubusercontent.com/1099479/41472099-ba96b692-70b5-11e8-8acd-10e50c942f54.png)